include/Model/Languages.php

```diff
@@ -1082,16 +1123,24 @@ protected function update_secondary_language_terms( $slug, $name, ?PLL_Language
 
 			if ( empty( $term_id ) ) {
 				// Attempt to repair the language if a term has been deleted by a database cleaning tool.
-				wp_insert_term( $name, $object->get_tax_language(), array( 'slug' => $slug ) );
+				$result = wp_insert_term( $name, $object->get_tax_language(), array( 'slug' => $slug ) );
+				if ( is_wp_error( $result ) ) {
+					$errors->add( 'pll_add_secondary_language_terms', $result->get_error_message() );
```

Comment:
To copy the error, you can use [`merge_from()`](https://developer.wordpress.org/reference/classes/wp_error/merge_from/). Otherwise here you are losing the error data if any (likely as if our code is well written, this a db insert error.